lib/vmpooler/api/v1.rb

```diff
@@ -747,6 +803,58 @@ def invalid_templates(payload)
 
       JSON.pretty_generate(result)
     end
+
+    post "#{api_prefix}/config/poolsize/?" do
+      content_type :json
+      result = { 'ok' => false }
+
+      need_token! if Vmpooler::API.settings.config[:auth]
+
+      payload = JSON.parse(request.body.read)
+
+      if payload
+        invalid = invalid_template_or_size(payload)
+        if invalid.empty?
```

Comment:
this cancels the whole request when at least one pool or size is invalid. Not sure if that's what you intend to do, as the other option is the accept the ones that are valid.

Reply:
Yes, that's what I intended to do since that is the pattern followed with the vm endpoint and it keeps things simple. I'm open to handling these cases in a more complex fashion, but wanted to minimize initial complexity.